master/getting-started/bare-metal/bare-metal.md

```diff
@@ -476,14 +472,50 @@ outside the cluster:
    normal host endpoint policy).  In other words, if a host endpoint is defined
    but has no pre-DNAT policies that explicitly allow or deny a particular
    incoming packet, that packet is allowed to continue on its way, and will
-   then be accepted or dropped according to workload policy (if it is going to
-   a local workload) or to normal host endpoint policy (if not).
+   then be accepted or dropped according to normal host endpoint policy applied on
+   forward traffic or a workload policy.
+
+## Policy on forward traffic
+If `applyOnForward` is `false`, host endpoint policy applies to traffic to/from local process only. 
+
+If `applyOnForward` is `true`, host endpoint policy applies to forward traffic include:
+1. Incoming traffic from a host endpoint to a local workload (container/pod/VM).
+2. Outgoing traffic from a local workload to a host endpoint.
+3. Traffic from one host endpoint that is forward to another host endpoint.
+
+By default, `applyOnForward` is `false`. 
+
+Untracked policy and Pre-DNAT policy must have `applyOnForward` set to `true` because it applies to all forwarded traffic. 
+ 
+Policies created before calico v2.7.0 will have `applyOnForward` set to default value `false` which means host endpoint polices and implicit default deny rules no longer apply to forward traffic by default. If you want a policy to apply forwarded traffic then you must explicitly set the `applyOnForward` to `true`. 
+
+> If you upgrade from a lower version of calico to v2.7.0 and
```

Comment:
Here again, you should use the new note style ``> **Note**: <your-note-text> {: .alert .alert-info}``